workbench-core/authentication/src/authenticationMiddleware.test.ts

```diff
@@ -35,6 +38,8 @@ const defaultCookieOpts = {
   sameSite: 'strict'
 } as const;
 
+const csrfToken = 'csrfToken';
+
 describe('authenticationMiddleware integration tests', () => {
```

Comment:
Do you have a test that verifies expected behavior if a csurf token is expected but not included in the request?

Reply:
That test should exist but is better suited for integration testing within `workbench-core/example` once that gets implemented.